src/test/java/org/broadinstitute/hellbender/tools/spark/sv/discovery/inference/AssemblyBasedSVDiscoveryTestDataProvider.java

```diff
@@ -0,0 +1,217 @@
+package org.broadinstitute.hellbender.tools.spark.sv.discovery.inference;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.variant.variantcontext.Allele;
+import htsjdk.variant.variantcontext.VariantContext;
+import htsjdk.variant.variantcontext.VariantContextBuilder;
+import htsjdk.variant.vcf.VCFConstants;
+import org.broadinstitute.hellbender.engine.datasources.ReferenceMultiSource;
+import org.broadinstitute.hellbender.exceptions.GATKException;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.BreakEndVariantType;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.SimpleSVType;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.SvType;
+import org.broadinstitute.hellbender.tools.spark.sv.discovery.alignment.AlignmentInterval;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+
+import static org.broadinstitute.hellbender.tools.spark.sv.utils.GATKSVVCFConstants.*;
+
+public abstract class AssemblyBasedSVDiscoveryTestDataProvider {
+
+    public abstract static class AssemblyBasedSVDiscoveryTestDataForSimpleChimera {
+        // these are used for building
+        public final AlignmentInterval firstAlignment;
+        public final AlignmentInterval secondAlignment;
+        public final String evidenceAssemblyContigName;
+        public final byte[] evidenceContigSeq;
+
+        // these are used for testing
+        public final boolean expectedFirstContigRegionHasLaterReferenceMapping;
+        public final SimpleChimera manuallyCuratedSimpleChimera;
```

Comment:
Can you describe what you mean by "manually curated"? Is it the just the expected values for these results? If so  can you rename them just to "expected"?